src/encoder/serde.rs

```diff
@@ -518,6 +533,17 @@ impl Serialize for JavaScriptCodeWithScope {
     }
 }
 
+impl Serialize for Binary {
```

Comment:
This follows the pattern of the other serializers defined here. Since the serialize implementation for Bson implements serializing from extjson, we piggyback off that to serialize Binary.